wled00/src/font/clear_3x5.h

```diff
@@ -0,0 +1,254 @@
+/**
+** The original 3x5 font is licensed under the 3-clause BSD license:
```

Comment:
Can you confirm that license conditions for this file are compatible with the MIT license of WLED? If not sure, we might need a UI element so user can opt-in to the font license terms before use.